extensions/lorem-ipsum/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "lorem-ipsum",
+  "title": "Lorem Ipsum",
+  "description": "Generate placeholder content",
+  "icon": "command-icon.png",
```

Comment:
Could you add a different icon? We don't allow usage of the default icon. You can replace it in the assets folder (size 512 X 512 pixels for the extension icon).